src/Fixer/Basic/BracesFixer.php

```diff
@@ -846,6 +846,11 @@ private function ensureWhitespaceAtIndexAndIndentMultilineComment(Tokens $tokens
 
         $nextToken = $tokens[$nextTokenIndex];
         if ($nextToken->isComment()) {
+            $previousToken = $tokens[$nextTokenIndex - 1];
+            // do not indent comments starting at the beginning of line - non-indented comments should not change indentation
+            if ($previousToken->isWhitespace() && 1 === preg_match('/\R$/', $previousToken->getContent())) {
+                return;
```

Comment:
@keradus For me not breaking code is more important than fixing it, and right now php-cs-fixer makes my code worse. I would understand this if such behavior will be result of separate fixer for indentation - then I can just disable it. But this is `braces` fixer, which is part of `PSR2` ruleset. For me, as a end user, this is completely unclear why `braces` fixer reindent my comments.